Tiltfile

```diff
@@ -68,6 +68,7 @@ k8s_yaml(kustomize('./deployment/local-dev'))
 allow_k8s_contexts('docker-desktop')
 
 lexbox_ports = [
+    port_forward(3000, name='ui'),
```

Comment:
sveltekit isn't running in the lexbox api pod, so I'm not sure this will do what you want, it should probably be added to the ui k8s resource.